PBN RCT.Rmd

```diff
@@ -523,18 +576,17 @@ pbn_complete %>%
   summarize(mean = mean(value),
             sd = sd(value))
 
-#N and percent of intervention participants with sum score >3, reflect overall perceived SSI acceptability
+#N and percent of intervention participants with sum score >3 (selects Agree and Strongly Agree only), reflect overall perceived SSI acceptability
 included_participants_pfs %>%
   filter(treatment_group == TRUE) %>%
-  mutate(pi_pfs_sum_scores = pi_pfs_sum / 7) %>%
-  mutate(pi_pfs_sum_scores_acceptable = pi_pfs_sum_scores > 3) %>%
-  summarise(
-    pi_pfs_sum_scores_acceptable_number = sum(pi_pfs_sum_scores_acceptable),
-    pi_pfs_sum_scores_acceptable_percent = (pi_pfs_sum_scores_acceptable_number * 100) / n()
-  )
+  mutate(pi_pfs_sum_scores = pi_pfs_sum / 7, 
+         pi_pfs_sum_scores_acceptable = pi_pfs_sum_scores > 3) %>%
+  summarise(pi_pfs_sum_scores_acceptable_percent = scales::percent(mean(pi_pfs_sum_scores_acceptable)))
 ```
 
-```{open-ended responses}
+```{r qual data}
+###COME BACK TO THIS### - Does not run right now because I removed text variables from  included_participants for imputation
```

Comment:
I would recommend having things go in this order

(1) clean and merge all data -> (2) save two clean datasets, one that's just the variables for quantitative analyses, and one that has the variables for qualitative analyses -> (3) use missforest to impute for the first dataset, but not the second

(1) is 99% of the data cleaning script; (2) should be like five lines; (3) is like 3 lines